cw-orch-daemon/src/state.rs

```diff
@@ -331,6 +332,19 @@ impl StateInterface for DaemonState {
     }
 }
 
+// copied from `tempfile::util::tmpname` implementation
+pub(crate) fn gen_temp_file_path() -> std::path::PathBuf {
```

Comment:
We use uid and tempfile crates in node tests

https://docs.rs/uid/latest/uid/
https://docs.rs/tempfile/latest/tempfile/

Do you prefer this setup?

Reply:
Yeah uid is be nice, thanks!
I'm not sure if we can use tempfile crate as it returns `File` object, which means we don't have control of creation for this object, but we need it for locking file, beyond that it's pretty hacky to get path from the `File` object

Reply:
Done, also noticed that uid not used (anymore?) in node tests, so removed unused part from there as well!